gin-client/git.go

```diff
@@ -69,13 +69,8 @@ func CommitIfNew() (bool, error) {
 // Setting the Workingdir package global affects the working directory in which the command is executed.
 func IsRepo() bool {
 	util.LogWrite("IsRepo '%s'?", Workingdir)
-	_, _, err := RunGitCommand("status")
+	_, _, err := RunGitCommand("rev-parse")
```

Comment:
Out of curiosity, is this documented behavior of rev-parse (like say --is-bare)?

Reply:
We just want a command that fails outside a git repository and succeeds inside a repository (even if it's bare). I was initially going to use something like `git rev-parse --show-toplevel` or `git rev-parse --git-dir`. The first doesn't work in bare repos but the second does.

It's not completely documented which commands work in bare repositories and which aren't, and the documentation doesn't specify what `rev-parse` with no arguments is supposed to do (a comment in the code simply states that it should just report whether CWD is in a git repo).

I suppose it would be clearer if we first did a `git status` check and failing that followed up with `git rev-parse --is-bare-repository`, but I like the simplicity of letting git just check if we're in a repository or not.